script.js

```diff
@@ -0,0 +1,113 @@
+const yearInput = document.getElementById("year-input");
```

Comment:
2000 is 21-st century

![Screenshot 2022-11-21 at 14 32 58](https://user-images.githubusercontent.com/34886027/203056541-093a18b7-f91b-44e6-a7d7-cd0d13b3545d.png)